ods_ci/tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -558,7 +558,7 @@ Fetch Image Tooltip Info
         ${item} =    Get Text    ${xpath_tooltip_items}\[${index}]
         Append To List    ${tmp_list}    ${item}
     END
-    Click Element    xpath://div[@class='pf-v5-c-popover__content']/button[@aria-label="Close"]
+    Click Element    xpath://div[@class='pf-v5-c-popover__close']/button[@aria-label="Close"]
```

Comment:
could you remove the "v5" and use `contains(@class, ..)` instead?

Reply:
Yeah, good call

Reply:
Done